### DISCUSSION TITLE
self-hosted Windows runners?

### DISCUSSION THREAD

**Discussion by caithaml** [dflook/terraform-github-actions#148] [Q&A] [answered]
Hi,
Is it possible to use self-hosted Windows runners? 
I always get this error when I try to run **terraform-plan action** on a self-hosted windows runner.
`Error: Container action is only supported on Linux`

**Comment by dflook** [comment, marked as answer]
As a docker based action, it is only available for linux runners 😞

**Reply by jrspinella** [reply]
Has anyone run self-hosted on GH Ent. getting a permission issue on the entrypoint on the apply run from the action
